lib/wasm/WebAssemblyGenerator.js

```diff
@@ -206,7 +206,7 @@ const rewriteImportedGlobals = state => bin => {
 
 				additionalInitCode.push(
 					/**
-					 * get_global in global initilizer only work for imported globals.
+					 * get_global in global initializer only works for imported globals.
 					 * They have the same indices than the init params, so use the
 					 * same index.
```

Comment:
@sokra can you clarify the context here :thinking:

Reply:
these globals are numbered in the same way as the init parameters are. So init parameter 1 matches global number 1.